src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Analysis/DistanceMeasurement/DistanceMeasurement.xaml.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+using Esri.ArcGISRuntime.UI.GeoAnalysis;
+using System;
+using System.Windows;
+using System.Windows.Input;
+
+namespace ArcGISRuntime.WPF.Samples.DistanceMeasurement
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Distance measurement analysis",
+        "Analysis",
+        "This sample demonstrates measuring 3D distances between two points in a scene. The distance measurement analysis allows you to add the same measuring experience found in ArcGIS Pro, City Engine, and the ArcGIS API for JavaScript to your app. You can set the unit system of measurement (metric or imperial) and have the units automatically switch to one appropriate for the current scale. The rendering is handled internally so they do not interfere with other analyses like viewsheds.",
+        "Choose 'New measurement' to start a new measurement. Move the mouse to change the end location. Tap to finish the measurement. Use the dropdown to select a different unit system.",
+        "Featured")]
+    public partial class DistanceMeasurement
+    {
+        // URLs to various services used to provide an interesting scene for the sample.
+        private readonly Uri _worldElevationService =
+            new Uri("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
```

Comment:
Are both of these elevation surfaces needed? It seems like the local one would be enough to illustrate the sample.

Reply:
This is the pattern followed by the sample on the other platforms its been implemented on.